### ISSUE TITLE
Issue with not closed attribute values

### ISSUE THREAD

**Issue by rseniuta** [htmllint/htmllint#166]
Having following config:
```
{
  "attr-name-style": false,
  "line-end-style": false,
  "id-class-style": false
}
```
this HTML:
`<div class="large id="title"></div>`

should throw error but does not.

**Comment by mlochbaum** [comment]
Yes, this is a problem. I suppose an `attr-close` rule, checking for proper quoting, is in order.

**Comment by mlochbaum** [comment]
Fixed by commit 9607be9. I called the option `attr-validate` since it just checks that the string between the tag name and closing angle bracket matches the spec (it should be `attr-valid` to describe what is required rather than what the parser does, but I just can't do that to grammar).

If the attributes are malformed in a way that prevents the parser from figuring out when they actually end, strange behavior may result. htmllint continues to not be an html validator.
